website/content/docs/security/acl/acl-rules.mdx

```diff
@@ -493,41 +503,30 @@ node "admin" {
 
-When reading from the catalog or retrieving information from the health endpoints, node rules are
-used to filter the results of the query. This allows for configurations where a token has access
-to a given service name, but only on an allowed subset of node names.
+Node rules are used to filter query results when reading from the catalog or retrieving information from the health endpoints. This allows for configurations where a token has access to a given service name, but only on an allowed subset of node names.
 
-Node rules come into play when using the [Agent API](/api/agent) to register node-level
-checks. The agent will check tokens locally as a check is registered, and Consul also performs
-periodic [anti-entropy](/docs/internals/anti-entropy) syncs, which may require an
-ACL token to complete. To accommodate this, Consul provides two methods of configuring ACL tokens
-to use for registration events:
+Consul agents check tokens locally when health checks are registered and when Consul performs periodic [anti-entropy](/docs/internals/anti-entropy) syncs.
+These actions may required an ACL token to complete. Use the following methods to configure ACL tokens for registration events:
 
-1. Using the [acl.tokens.default](/docs/agent/options#acl_tokens_default) configuration
-   directive. This allows a single token to be configured globally and used
-   during all check registration operations.
-2. Providing an ACL token with service and check definitions at
-   registration time. This allows for greater flexibility and enables the use
-   of multiple tokens on the same agent. Examples of what this looks like are
-   available for both [services](/docs/agent/services) and
-   [checks](/docs/agent/checks). Tokens may also be passed to the
-   [HTTP API](/api) for operations that require them.
+* Configure a global token in the [acl.tokens.default](/docs/agent/options#acl_tokens_default) parameter. 
+This allows a single token to be used during all check registration operations.
+* Provide an ACL token with service and check definitions at registration time. 
+This allows for greater flexibility and enables the use of multiple tokens on the same agent. 
+Refer to the [services](/docs/agent/services) and [checks](/docs/agent/checks) documentation for examples.
+Tokens may also be passed to the [HTTP API](/api) for operations that require them.
 
-In addition to ACLs, in Consul 0.9.0 and later, the agent must be configured with
-[`enable_script_checks`](/docs/agent/options#_enable_script_checks) set to `true` in order to enable
-script checks.
+
+-> **Script checks are required for Consul 0.9.0 and later**. In addition to ACLs, the agent must be configured with
+[`enable_script_checks`](/docs/agent/options#_enable_script_checks) set to `true` to enable
+script checks in Consul 0.9.0 and later0.
```

Comment:
I'm not entirely sure why this guidance is here. It seems like this should be documented at `/docs/agent/options#_enable_script_checks` instead of as part of the ACL rule reference.

Nothing about the previous paragraph seems related to ACL rules.

But I think this edit is unfortunately even more misleading. Script checks are not required for Consul 0.9.0.

Reply:
I revised and moved this guidance to the agent configuration page.